components/admin_console/system_user_detail/system_user_detail.scss

```diff
@@ -6,6 +8,20 @@
 }
 
 .SystemUserDetail__input {
+    display: inline-block;
     height: 40px;
     max-width: 320px;
-}
\ No newline at end of file
+}
+
+.SystemUserDetail__field-text {
+    vertical-align: middle;
+}
+
+.SystemUserDetail__field-icon {
+    width: 20px;
+    height: 20px;
+    fill: $dark-gray;
```

Comment:
@michaelgamble or @asaadmahmood Invision specifies `background-color: rgba(0,0,0,0.4);` but we've got a color variable `$dark-gray: rgba(0, 0, 0, .5);` in `_variables.scss`. Should we add `0.4` to the variables? Or is this a one-off case with the icon?

Reply:
@bradjcoughlin It should not use the $dark-gray variable, it should be set via the theme, or css3 variables. You can use:
`v(center-channel-color-40)` instead of `$dark-gray`.